app/controllers/api/newsfeed/coins_controller.rb

```diff
@@ -1,8 +1,12 @@
 class Api::Newsfeed::CoinsController < ApiController
 
   def index
-    @coins = Coin.order(:ranking).limit(20)
-    respond_success index_serializer(@coins)
+    coin_ids = Coin.order(:ranking).limit(20).ids
```

Comment:
it looks like you're hardcoding the `.limit(20` in the controller? is this exposed in the endpoint, so the user can choose to request 50 coins if they needed to?

Reply:
Yep 20 is what we want for now, easy to change though